tabbycat/api/views.py

```diff
@@ -440,12 +440,42 @@ class SubstantiveSpeakerStandingsView(BaseStandingsView):
     def get_max_round(self):
         return self.tournament.round_set.last()
 
+class SubstantiveSpeakerStandingsRoundsView(TournamentAPIMixin, TournamentPublicAPIMixin, ModelViewSet):
+    serializer_class = serializers.SpeakerRoundsSerializer
+    tournament_field = "team__tournament"
+    access_preference = 'public_participants'
```

Comment:
It's not enough to have access to the participants list to get the tab. The logic may be a bit more complicated, but it should allow viewing with `speaker_tab_released` or for completed rounds that aren't silent.